homeassistant/components/template/button.py

```diff
@@ -135,5 +131,5 @@ def __init__(
 
     async def async_press(self) -> None:
         """Press the button."""
-        if self._command_press:
-            await self.async_run_script(self._command_press, context=self._context)
+        if (script := self._action_scripts.get(CONF_PRESS)) is not None:
```

Comment:
The `_action_scripts` dict can only have `Script` objects, the specific None check is not needed. Same comment for other classes.
```suggestion
        if script := self._action_scripts.get(CONF_PRESS):
```